scripts/reconfigure.mjs

```diff
@@ -1,7 +1,7 @@
-import { writeFile } from '@morio/shared/fs'
-import { resolveServiceConfiguration, getPreset } from '@morio/config'
-import { pullConfig } from '@morio/config'
-import { Store } from '@morio/shared/store'
+import { writeFile } from '@itsmorio/shared/fs'
```

Comment:
These `@itsmorio` prefix will break this script. Please restore the version from `develop`